README.md

```diff
@@ -9,3 +13,12 @@ Documentation of energetically autonomous and highly dynamic anthropomorphic rob
 2. Clone the repository;
 3. Run `python -m mkdocs serve` in the project root. The changes you make in the
    documentation should be visible in `http://localhost:8000/`;
+
+## Contributing
```

Comment:
Thanks! I think it would be better to move the `Contributing` section and detail it a bit more as we discussed.

Reply:
I added the missing content and the License section. However, it is still such little content that I'd rather leave it in this file. What do you think?